### PR TITLE
Config with default template

### PR DESCRIPTION
**Pull request by gforcada**

### CODE REVIEW

**Comment by gforcada** [comment]
@jenkins-plone-org please run jobs

File: setup.cfg [hunk 1]
```diff
@@ -1,17 +1,23 @@
-[build_sphinx]
-source-dir = docs/source
-build-dir  = docs
-all_files  = 1
-
-[upload_sphinx]
-upload-dir = docs/html
```
Comment (mauritsvanrees) [inline setup.cfg:7]: We may need to keep the sphinx configuration. :-/ It seems used by the Plone 5 documentation:
https://docs.plone.org/external/plone.app.caching/docs/index.html

Is there any way to keep this with current meta? We could put it back by hand for now.

**Comment by gforcada** [comment]
Note that when this is merged the repository needs to be added on the [linting global required workflow](https://github.com/organizations/plone/settings/actions) 🗒️

File: setup.cfg (hunk 1 above)
Comment (mauritsvanrees) [reply setup.cfg:7]: I re-added it manually for now.

**Review by mauritsvanrees — APPROVED** [review]

**Comment by mauritsvanrees** [comment]
Let's first get this merged, before doing another update from plone/meta. But first Jenkins:

@jenkins-plone-org please run jobs

**Comment by mauritsvanrees** [comment]
> Note that when this is merged the repository needs to be added on the [linting global required workflow](https://github.com/organizations/plone/settings/actions) 🗒️

Done.

**Pull request merged**